dashboard/lib/routes/index.js

```diff
@@ -62,7 +63,7 @@ function get_index (req, res) {
 
                 logger.debug('before render: %s', JSON.stringify(regions));
                 console.log({name: userinfo.displayName, regions: regions, role: req.session.role});
-                res.render('logged', {name: userinfo.displayName, regions: regions, role: req.session.role});
+                res.render('logged', {name: userinfo.displayName, regions: regions, role: req.session.role,logout_url: config.idm.logoutURL});
```

Comment:
mmm, this line has more than 120 characters and I suggest to separate the "req.session.role,logout_url" in "req.session.role, logout_url..."